preprocessing/nextclade/src/loculus_preprocessing/processing_functions.py

```diff
@@ -816,12 +816,32 @@ def check_authors(
                     warnings=warnings,
                     errors=errors,
                 )
+            if " and " in authors:
+                warning_message = (
+                    f"Authors list '{authors}' contains 'and'. This may indicate a misformatted authors list. Authors should always be separated by semi-colons only e.g. `Smith, Anna; Perez, Tom J.; Xu, X.L.`."
+                )
+                warnings = [
```

Comment:
we should actually append these warnings (also the one above to the already existing warning list so we can warn about multiple possibly invalid authors)